front/app/containers/Admin/users/UserTableActions.tsx

```diff
@@ -157,6 +157,7 @@ interface Props {
   allUsersIds: string[];
   groupId?: string;
   deleteUsersFromGroup?: (userIds: string[]) => void;
+  search: string | null;
```

Comment:
In some places we're calling the property `searchTerm`, and others it's `search` - should we try and stick with one version? Or is there a specific reason why it's different name? :)